docs/pages/platform/schema-validation.mdx

```diff
@@ -54,10 +45,16 @@ You can interact with the schemas you create through the dashboard or by calling
 the [REST API](/docs/api-reference/rest-api-endpoints). In this guide, we will
 cover the following operations:
 
-- [Creating a schema](#Creating-a-schema)
-- [Deleting a schema](#Deleting-a-schema)
-- [Attaching a schema to a room](#Attaching-a-schema-to-a-room)
-- [Detaching a schema from a room](#Detaching-a-schema-from-a-room)
+- [Why schema validation?](#why-schema-validation)
+- [How schema validation works](#how-schema-validation-works)
+- [Schema actions](#schema-actions)
+  - [Creating a schema](#creating-a-schema)
+  - [Deleting a schema](#deleting-a-schema)
+  - [Attaching a schema to a room](#attaching-a-schema-to-a-room)
+  - [Detaching a schema from a room](#detaching-a-schema-from-a-room)
+- [The validation process in practice](#the-validation-process-in-practice)
+- [How to handle validation errors](#how-to-handle-validation-errors)
+- [What’s to come](#whats-to-come)
```

Comment:
```suggestion
- [Creating a schema](#creating-a-schema)
- [Deleting a schema](#deleting-a-schema)
- [Attaching a schema to a room](#attaching-a-schema-to-a-room)
- [Detaching a schema from a room](#detaching-a-schema-from-a-room)
```

I think we'll revert this for now, as it's specifically showing you the 4 schema operations (all the other links are automatically available in the sidebar).

Reply:
Oh!! Good catch! That's my Visual Studio which automatically added it!!

Thanks!